package.json

```diff
@@ -45,7 +45,8 @@
     "react-native": "*"
   },
   "dependencies": {
-    "lodash": "4.x.x"
+    "lodash": "4.x.x",
+    "prop-types": "^15.6.0"
```

Comment:
this should not be a dependency, it should go into devDependencies and peerDependencies: and with "version *"